.github/workflows/inspektor-gadget.yml

```diff
@@ -1,6 +1,8 @@
 name: Inspektor Gadget CI
 env:
-  CONTAINER_REPO: ghcr.io/${{ github.repository }}
+  REGISTRY: ghcr.io
+  RELEASE_CONTAINER_REPO: ${{ github.repository }}
+  DEVELOPMENT_CONTAINER_REPO: ${{ github.repository }}-dev
```

Comment:
Wasn't it `-ci`? 

BTW, could you please also add a `-` in the `$(CONTAINER_REPO)test` so that we have:
- `ghcr.io/kinvolk/inspektor-gadget`
- `ghcr.io/kinvolk/inspektor-gadget-ci`
- `ghcr.io/kinvolk/inspektor-gadget-test`

If you prefer, you can do it in another PR.